experimental/dds/tree2/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -185,8 +190,8 @@ export class SchemaEditor<TRepository extends StoredSchemaRepository>
 		const op: JsonCompatibleReadOnly = content;
 		if (isJsonObject(op) && op.type === "SchemaOp") {
 			assert(
-				typeof op.data === "string",
-				0x5e3 /* expected string data for resubmitted schema op */,
+				this.formatValidator.check(op.data),
```

Comment:
@Abe27342, is this necessary in your opinion? This is a replacement for the prior behavior that validated that `op.data` is a string. But arguably, that validation didn't do a whole lot anyway, unless there's some buggy history I'm not aware of.

Reply:
Eh, you could make an argument either way. Depends on if we want to trust the runtime or not to not muck with the op contents between submission and resubmission. It seems fine to keep around (and it's nice that it actually validates something more meaningful now)

Reply:
Cool, my only worry was that it might be slow or something, but I don't expect it will be.

Reply:
Schema is unlikely to be the dominating factor on any validation that's done of persisted format (for long-lived documents, generally forest will take the most time, maybe edit manager as well).

I did profile impact of using typebox's validator on some of our e2e tests. It came out to about 10-20% overhead while loading the summary, which isn't a crazy pill to swallow if trying to be super safe about avoiding doc corruption.
The nice part about making this part of shared-tree's config is that it's up to the application if they want to pay this cost. They could even make more interesting policies like pay it only in pre-prod environment and tests but no validation in production, if they wanted.

Reply:
Awesome, I'm not worried at all then. Thanks for the details.